src/components/top-bar/styles.ts

```diff
@@ -1,11 +1,9 @@
-import { css } from '@linaria/core'
 import { styled } from '@linaria/react'
-import { isDesktopOrBelow, isMobile, isTabletOrBelow } from '../../styles/media'
+import { isDesktop, isDesktopOrBelow, isTablet, isTabletOrBelow } from '../../styles/media'
 import { ElButtonGroup } from '../button-group'
-import { ElNavSearchButton } from '../nav-search-button/styles'
-import { ElButtonNavIconItem } from '../nav-icon-item'
+import { AvatarButton } from '../avatar-button'
 
-export const elTopBarAvatar = css`
+export const ElTopBarProfile = styled(AvatarButton)`
```

Comment:
Ah yes, I was a little confused because TopBar.Search previously had multiple children. However, after updating it to a single child, I shouldn't have missed it 🤦 .